### PR TITLE
Added Function for Saving

### LINKED ISSUE #26: Mask PNG files do not appear easily in standard image viewers
I guess now that the mask pixels are valued according to user input `[0, 1, 2]`, they don't nicely show up as white in the `.png` file. Can a second `.png` file be saved, with instead a white background and mask colours corresponding to the program?

### LINKED ISSUE #27: File overwrite warning not implemented in `annotate_img`
Seems `.txt` and `.png` are automatically overwritten now without prompting.

### PR DESCRIPTION
**Pull request by msekatchev**
closes #27 
- Created `saveFile` function for overwrite testing, called from `annotate_img` and `annotate_dir`
closes #26 
- Now creates visible image mask in addition to standard `[0,1,2]` mask

### CODE REVIEW

File: annotate.py [hunk 1]
```diff
@@ -3,15 +3,20 @@
 import numpy as np
 import shutil
 
+#annotate_img("test/B.jpg","MS",10,120)
```
Comment (pdeperio) [inline annotate.py]: Why did this come back?

File: annotate.py [hunk 2]
```diff
@@ -25,54 +30,47 @@ def draw_circle(event,x,y,flags,param):
...
-#    {image_name}.txt - A text file. Each line contains a PMT feature ID and its pixel coordinates on the image. The text file has the same name as the image.
-
-##User keypresses (make sure the image window is selected):
-# click on image to grab pixel coordinates:
-#	 Left-click to label with red 
-#	 middle-click to label with blue
-# s to close image and exit program
-# r to write coordinates (you will be prompted for the PMT feature ID if a PMT has not been selected yet)
-# f to select new PMT
-# n to increment the featureID number
```
Comment (pdeperio) [inline annotate.py:71]: Lost documentation of incrementing?

File: annotate.py [hunk 3]
```diff
@@ -25,54 +30,47 @@ def draw_circle(event,x,y,flags,param):
...
-# b to decrement the featureID number
+##function: annotate_img
+    ##inputs
+        ##img_path - name and location of the image, example "B.jpg"
+        ##size1 - size of the 1st brush circle, in px
+        ##size2 - size of the 2nd brush circle, in px
+        ##initials - First name and last name initials of the person doing the labelling. Recorded next to every PMT feature ID line in the text file.
+    ##outputs
+        ##label.jpg - A map of all the features in binary (white on black)
```
Comment (pdeperio) [inline annotate.py]: Seems like older version of documentation got injected here?

**Review by pdeperio — CHANGES_REQUESTED** [review]
Sorry for delay, slipped my mind, thanks for reminder. 

Just some small comments on documentation.

File: annotate.py (hunk 3 above)
Comment (msekatchev) [reply annotate.py]: Hmm yeah can't remember how that happened lol
Sorry about that I'll make a new commit

File: annotate.py (hunk 1 above)
Comment (msekatchev) [reply annotate.py]: Sorry about that, I put that there for easy access and forgot to remove it

Comment (pdeperio) [reply annotate.py]: is it not stored in your command history (when you press ↑ in terminal)?

Comment (msekatchev) [reply annotate.py]: True, the way I have it set up with Sublime Text a new terminal runs every time so the commands are refreshed...
Not the most efficient, but on the other hand I can quickly run the code with a shortcut.

**Review by pdeperio — APPROVED** [review]

**Pull request merged**